### DISCUSSION TITLE
Add a license to changelog.md itself

### DISCUSSION THREAD

**Discussion by marcoXbresciani** [olivierlacan/keep-a-changelog#516] [Ideas]
Following GNU project / FSF requirements I received for my projects on Savannah (NonGNU) software forgery (https://savannah.nongnu.org/users/marcobresciani) I think it should be nice to add a License concept for the CHANGELOG.md file itself.

1. There should be a license for the CHANGELOG.md as well, for example the GNU All-Permissive License (see https://www.gnu.org/licenses/license-list.en.html#GNUAllPermissive) or the GNU FDL (see https://www.gnu.org/licenses/fdl.html) if the file itself is longer than 300 lines, as suggested by FSF (see https://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html), so that the file itself has a license.

2. If the GNU FDL is used, a "GNU Free Documentation License" section is needed according to the license itself (see https://www.gnu.org/licenses/fdl.html#addendum: 

> A copy of the license is included in the section entitled "GNU Free Documentation License".

).

Something like this might be nice:
```markdown
<!--
Copyright ©  YEAR(S)  Author(s).

Copying and distribution of this file, with or without modification,
are permitted in any medium without royalty provided the copyright
notice and this notice are preserved.  This file is offered as-is,
without any warranty.
-->
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

     ,' ,-_-. '.
    ((_/)o o(\_))
     `-'(. .)`-'
         \_/

    Copyright ©  YEAR(S)  Author(s).

    Copying and distribution of this file, with or without modification,
    are permitted in any medium without royalty provided the copyright
    notice and this notice are preserved.  This file is offered as-is,
    without any warranty.
```
with also the comment header containing the license similar to copyright notices in source files.

**Comment by FedericoVaga** [comment]
This could be useful if you are using [REUSE](https://reuse.software/) and you run their tool for checking that you are properly licensing your sources.

**Comment by FedericoVaga** [comment]
One more comment. Instead of the boilerplate it should be recommended to use SPDX tags.

**Comment by marcoXbresciani** [comment]
> One more comment. Instead of the boilerplate it should be recommended to use SPDX tags.

Might be, with the "GNU All-Permissive License" I've put in the example, but if you're licensing the CHANGELOG.md with, say, GNU FDL, you're required to put the license in the document itself.

> A copy of the license is included in the section entitled "GNU Free Documentation License".

So, to be compliant with such license, you have to include it:

> To use this License in a document you have written, include a copy of the License in the document and put the following copyright and license notices just after the title page:
>>    Copyright (C)  YEAR  YOUR NAME.
>>    Permission is granted to copy, distribute and/or modify this document
>>    under the terms of the GNU Free Documentation License, Version 1.3
>>    or any later version published by the Free Software Foundation;
>>    with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts.
>>    A copy of the license is included in the section entitled "GNU
>>    Free Documentation License".
